stackrox/base/externalsecrets/central-htpasswd.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+metadata:
+    name: central-htpasswd
+spec:
+    dataFrom:
+        - extract:
+              key: moc/smaug/stackrox/central-htpasswd
```

Comment:
Putting something in base means that, it would be applied to deploy it in any of our deployments. This could be an issue if we want to deploy stackrox on another cluster. For this, I would patch this value in the overlay kustomization or I would just put the whole externalsecret itself in the smaug overlay.

Reply:
see example for how you would patch: https://github.com/operate-first/apps/blob/master/kafka/overlays/balrog/kustomization.yaml